pymrio/tools/iomath.py

```diff
@@ -410,3 +410,44 @@ def calc_accounts(S, L, Y, nr_sectors):
     )
 
     return (D_cba, D_pba, D_imp, D_exp)
+
+
+def calc_trade_flows(Z, Y):
+    """ Calculate the bilateral trade flows from the Z and Y matrix
+
+    Parameters
+    ----------
+    Z : pandas.DataFrame
+        Symmetric input output table (flows)
+    Y : pandas.DataFrame
+        final demand with categories (1.order) for each country (2.order)
+
+    Returns
+    -------
+    pandas.DataFrame
+        bilateral trade flows as a matrix showing region and sector of origin in rows, and region of import in columns
+                
+   NOTE: ONLY IMPLEMENTED FOR DataFrame right now
```

Comment:
In numpy docstrings (which you are using here, thats great), the notes would come before the Parameters like:

Notes
---------

ONLY IMPLEMENTED FOR DataFrame right now

Parameters
---------------
...

small detail, if you want to change.

Reply:
done - hope that is OK!